doonut-app-external-api/src/main/kotlin/com/doonutmate/example/controller/ExampleController.kt

```diff
@@ -2,20 +2,35 @@ package com.doonutmate.example.controller
 
 import com.doonutmate.example.controller.dto.ExampleCreateReq
 import com.doonutmate.example.service.ExampleAppService
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.media.ArraySchema
+import io.swagger.v3.oas.annotations.media.Content
+import io.swagger.v3.oas.annotations.media.Schema
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.responses.ApiResponses
+import io.swagger.v3.oas.annotations.tags.Tag
 import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.web.bind.annotation.PostMapping
 import org.springframework.web.bind.annotation.RequestBody
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
 
 @RestController
+@Tag(name = "example", description = "테스트용 API")
 @RequestMapping("/example")
 class ExampleController(
     @Autowired var exampleAppservice: ExampleAppService,
 ) {
-
     @PostMapping
+    @Operation(summary = "테스트용 API", description = "=====API 설명=====")
+    @ApiResponses(
+        value = [
+            ApiResponse(responseCode = "200", description = "성공", content = [(Content(mediaType = "application/json", array = (ArraySchema(schema = Schema(implementation = ExampleCreateReq::class)))))]),
+            ApiResponse(responseCode = "400", description = "Bad Request", content = [Content()]),
+            ApiResponse(responseCode = "404", description = "Not Found", content = [Content()]),
+        ],
+    )
     fun create(@RequestBody req: ExampleCreateReq) {
```

Comment:
이건 취향차이긴 한데 전 `@PostMapping` 같은 애노테이션은 메서드에 최대한 붙여놓고, `@Operation`, `@ApiResponses` 같은 문서화에 필요한 애노테이션은 메서드에 멀리 떨어져놓는 편이에요. 문서화 툴은 나중에 바뀔 수도 있고, 비즈니스로직에 직접적으로 영향을 주지 않기 때문입니다.

```suggestion
    @Operation(summary = "테스트용 API", description = "=====API 설명=====")
    @ApiResponses(
        value = [
            ApiResponse(responseCode = "200", description = "성공", content = [(Content(mediaType = "application/json", array = (ArraySchema(schema = Schema(implementation = ExampleCreateReq::class)))))]),
            ApiResponse(responseCode = "400", description = "Bad Request", content = [Content()]),
            ApiResponse(responseCode = "404", description = "Not Found", content = [Content()]),
        ],
    )
    @PostMapping
    fun create(@RequestBody req: ExampleCreateReq) {
```

Reply:
저도 이 부분 동의합니다!

Reply:
어노테이션 위치에 관한 부분은 고민을 하지 않고 익숙한대로 작성해왔었는데, 스웨거 어노테이션 처럼 직접적으로 비즈니스 로직에 영향을 주지 않는 코드는 멀리 떨어트려 놓는게 맞다는 생각이 듭니다. `@Operation` 이나 `'@ApiResponses`처럼 어는 정도 크기가 있는 어노테이션은 비즈니스 로직 근처에 붙여두니 코드의 가독성을 해치는것 같다는 생각을 하였습니다!!